langchain4j-chroma/src/main/java/dev/langchain4j/store/embedding/chroma/ChromaEmbeddingStore.java

```diff
@@ -171,6 +171,13 @@ public List<EmbeddingMatch<TextSegment>> findRelevant(Embedding referenceEmbeddi
                 .collect(toList());
     }
 
+    @Override
+    public List<EmbeddingMatch<TextSegment>> findRelevant(Object memoryId, String originText, Embedding referenceEmbedding, int maxResults, double minScore) {
+        return findRelevant(referenceEmbedding,maxResults,minScore);
```

Comment:
This logic is repeating, can you put it as a default method in `EmbeddingStore` interface?

Reply:
yes  i would add a default interface in EmbeddingStore